### ISSUE TITLE
Wallace Memorial Fund site is loading extremely slowly.

### ISSUE THREAD

**Issue by Hyposcada** [NaturalHistoryMuseum/scratchpads2#6497]
The Wallace Memorial Fund site (https://wallacefund.myspecies.info/) is loading extremely slowly, if at all..

**Comment by therobyouknow** [comment]
Good morning @Hyposcada sorry to hear about this, can you advise on any particular pages that are slow.


It does seem to be working better now, from what I've just checked:
- https://wallacefund.myspecies.info/ - loads swiftly
- https://wallacefund.myspecies.info/sites/wallacefund.myspecies.info/files/Beccaloni.2020.PortraitsOfWallace.Version3.pdf - fine also
- https://wallacefund.myspecies.info/news_views - fine 
- https://wallacefund.myspecies.info/biblio - fine

When you visited it and found it to be slow, was this the first time you had visited it recently?  I'm wondering if the cache needed to be warmed up, which meant that when I visited it, it ran faster. However, I'd imagine the site could have had many visitors recently, so that theory I have may not carry much weight. Rather, it could be an intermittent server performance problem at the time you visited when you found it to be slow.

Certainly please let me know your thoughts on how the site is functioning currently.

**Comment by Hyposcada** [comment]
Hi Rob (I'm guessing),

Thanks for your email and for checking the site. It was not loading at all
over the weekend, and it has been painfully slow for the past two days.
This morning, however, it seems fine! I will let you know if it grinds to a
halt again. This has actually been a problem from time to time over the
years, and I have no idea what causes it, but I'm sure it isn't my computer
or internet connection.

Happy Christmas!

George
[image: Christmas2021.jpg]

****************************************************************************
*George Beccaloni, PhD*
*Director, Alfred Russel Wallace Correspondence Project*
Other email addresses: ***@***.***

*My websites*:
Wallace Correspondence Project: https://wallaceletters.myspecies.info
Catalogue of Wallace's correspondence: https://tinyurl.com/5butz9pe
Wallace Fund Website: https://wallacefund.myspecies.info
Cockroach Species File Online: http://cockroach.speciesfile.org
LepIndex: *https://www.nhm.ac.uk/our-science/data/lepindex/
<https://www.nhm.ac.uk/our-science/data/lepindex/>*
HOSTS: https://www.nhm.ac.uk/our-science/data/hostplants

ResearchGate: https://www.researchgate.net/profile/George_Beccaloni
****************************************************************************

**Comment by therobyouknow** [comment]
Thank you George @Hyposcada pleased it's working well for you at the moment, thank you for advising.

Performance issues are definitely something that needs addressing across the sites as the other issues in github indicate, and I appreciate reports of issues anytime, please  certainly update if any change. Information about issues including when they happen can help as we analyse the issues and find root causes to fix.

Thanks again, and Happy Christmas! :) Rob

**Comment by therobyouknow** [comment]
Good morning @Hyposcada how are you finding this now?

I've looked on https://wallacefund.myspecies.info/ and am finding it to have good responsiveness.

I'm hoping that our recent server upgrades and related fixes will have improved the performance. 

https://github.com/NaturalHistoryMuseum/scratchpads2/issues/6505

From above observations of your site, there are encouraging early signs - as other admins of other sites are also experiencing improved performance.

Please feedback here when you can.  If you are finding the site is running well, I will close this ticket for the time being, but we will continue to monitor performance and always respond to issues here.

**Comment by Hyposcada** [comment]
Yes, it is a lot more responsive. Thanks very much for sorting it out. This has been a problem for c. 2 years.

**Comment by therobyouknow** [comment]
Thank you for your feedback @Hyposcada . I am pleased to hear that it is a lot more responsive - the outcome we were hoping for with our upgrade and associated fixes.

Closing ticket.
